static/js/subs.js

```diff
@@ -966,7 +966,9 @@ exports.initialize = function () {
         $(".right").removeClass("show");
         $(".subscriptions-header").removeClass("slide-left");
     });
-
+    $("#subscriptions_table").on("click", ".exit", () => {
+        $(".colorpicker").spectrum("hide");
```

Comment:
How did you choose `hide` vs. `destroy` as the event to send?  We appear to use `destroy` in the left sidebar.

Also, please follow the existing whitespace conventions of a blank line between blocks.

Reply:
@timabbott  I used hide as it not remove all color picker functionalities and restore the element.

Reply:
What would happen if we used `destroy` instead?  Would that be a bug?

Reply:
As you requested to use destroy  I changed  $(".colorpicker").spectrum("hide") to $(".colorpicker").spectrum("destroy").It work fine. No bug.

Reply:
There is no bug as the logic for render color picker as follows:
1) if already there is color picker functionalities it simply call show.
2) else it create all color picker functionalities and call show.

Who implement the color picker, handled both cases carefully. Therefore no bug.